pkgs/create-neon/data/versions.json

```diff
@@ -1,5 +1,4 @@
 {
-  "neon": "1.0.0",
-  "napi": "8",
+  "neon": "1",
```

Comment:
This felt cleaner because `1.0.0` and `1` mean the same thing with any additional specifiers.